src/patternfly/components/Alert/examples/danger-alert.hbs

```diff
@@ -1 +1,33 @@
-{{#> alert modifierClass="pf-m-danger"}}<p>Lorem ipsum dolor sit amet, consectetur adipisicing elit.</p>{{/alert}}
\ No newline at end of file
+{{#> alert pf-c-alert--Modifier="pf-m-danger pf-m-top-aligned" pf-c-alert--AriaLabel="Danger Notification"}}
+<div aria-hidden="true" class="pf-c-alert__icon">
+  <i class="fas fa-exclamation-circle"></i>
+</div>
+<div class="pf-c-alert__body">
+  <h4 class="pf-c-alert__title">
+    <span class="sr-only">Danger: </span> Danger notification title</h4>
```

Comment:
Our current documentation for [toast notifications](http://www.patternfly.org/pattern-library/communication/toast-notifications/#design) and [inline notifications](http://www.patternfly.org/pattern-library/communication/inline-notifications/#design) never actually refers to this type as 'danger' but calls this type 'error' instead. 

@mcarrano: Is 'danger' a term that we actually want to surface in the UI, or is 'error' what you'd expect here?

Reply:
ya looks like we have a misalignment between the docs and the code. We use the class `.alert-danger` in core. @kybaker uses Critical in the designs too so maybe we need to consider what we want to call this.